flytekit/remote/execution.py

```diff
@@ -0,0 +1,105 @@
+from dataclasses import dataclass
+from typing import Any, Dict, Union, List
+
+from flytekit import ContainerTask
+from flytekit.annotated.context_manager import SerializationSettings
+from flytekit.annotated.promise import translate_inputs_to_literals
+from flytekit.annotated.reference_entity import ReferenceEntity
+from flytekit.common.core.identifier import WorkflowExecutionIdentifier
+from flytekit.common.exceptions import user as user_exceptions
+from flytekit.common.translator import get_serializable
+from flytekit.common.workflow_execution import SdkWorkflowExecution
+from flytekit.models.common import Notification, Annotations, Labels, AuthRole
+from flytekit.models.execution import ExecutionSpec, ExecutionMetadata, NotificationList
+from flytekit.clients.friendly import SynchronousFlyteClient
+from flytekit.models.literals import LiteralMap
+from flytekit.annotated import context_manager as flyte_context
+from flytekit.models.task import TaskSpec
+from flytekit.configuration import auth as auth_config
+
+
+@dataclass
+class ExecutionParameters(object):
+    notifications: List[Notification] = None
+    labels: Labels = None
+    annotations: Annotations = None
+    auth_role: AuthRole = None
+    # optional name to assign to execution
+    name: str = None
+    # The name of the user launching this execution.
+    principal: str = None
+
+
+def execute(
+    entity: Union[ReferenceEntity, ContainerTask],
+    client: SynchronousFlyteClient,
```

Comment:
can this be optional?

Reply:
sure, we can always initialize using the platform url in the config